src/python/README.md

```diff
@@ -104,4 +104,5 @@ With this workflow, you can avoid having to stop each service after client reque
     - [Metrics](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md)
     - VertexAI
     - Sagemaker
-    - [Restricted Protocols](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/inference_protocols.md#limit-endpoint-access-beta)
\ No newline at end of file
+    - [Restricted Protocols](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/inference_protocols.md#limit-endpoint-access-beta)
+    - After a running server has been stopped, if the client sends an inference request, it will result in a Segmentation Fault.
```

Comment:
Fix indentation, this shouldn't be at the same level as the "unsupported features", it should be at top level
```suggestion
- After a running server has been stopped, if the client sends an inference request, it will result in a Segmentation Fault.
```